aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -104,18 +104,18 @@ Tensor PerTensorAffineQuantizer::quantize(Tensor rtensor) {
   // quantizer that can be reused, so I'm using intrusive_from_this here
   Tensor qtensor = new_qtensor(
       rtensor.sizes(),
-      rtensor.options().dtype(scalar_type_),
+      rtensor.options().dtype(scalar_type_).memory_format(rtensor.suggest_memory_format()),
```

Comment:
do we also need to make qtensor contiguous then? or it is created as a contiguous tensor

Reply:
I think by default it does that. I remember new_qtensor does refresh_contiguous, so I am assuming it makes it contiguous in the memory format it is allocated in.